install

```diff
@@ -12,36 +12,34 @@ command -v git > /dev/null || {
 	exit 2;
 }
 
+# Get location where bro will be installed
+read -p "BRO_STATION? ($HOME/.bro):" BRO_STATION
+if [ "$BRO_STATION" = ""  ]; then
+	BRO_STATION="$HOME/.bro"
+fi
```

Comment:
This might result in multiple installation of bro. If bro is already installed its obvious `BRO_STATION` will be available in shell environment. So I think we should check if bro is already installed before asking installation path for bro. :)